src/frontend/src/index.ts

```diff
@@ -32,7 +33,30 @@ const readCanisterId = (): string => {
   return setupJs.dataset.canisterId;
 };
 
+// Show version information for the curious programmer
+const printVersion = () => {
+  console.log("Welcome to Internet Identity!");
+  console.log(
+    "The code can be found here: https://github.com/dfinity/internet-identity"
+  );
+  console.log(
+    `https://github.com/dfinity/internet-identity/commit/${version.commit}`
+  );
+  if (version.release !== undefined) {
+    console.log(`This is version ${version.release}`);
+  }
+  if (version.dirty) {
+    console.warn("This version is dirty");
+  }
+};
+
 const init = async () => {
+  try {
+    printVersion();
+  } catch (e) {
+    console.warn("Error when printing version information:", e);
```

Comment:
not sure, if printVersion fails it's most probably for some weird reason because console.something would fails so here it would probably fails again no? I would just silent it or remove the try catch

Reply:
Yeah but if for _some_ reason e.g. `version.commit` cannot be printed or something, I don't want the whole app to fail because of this... just being extra cautious here since if anything breaks we wouldn't even show an error message

Reply:
Sounds good